Source/WebCore/page/scrolling/ScrollingStateNode.h

```diff
@@ -225,37 +225,38 @@ enum class ScrollingStateNodeProperty : uint64_t {
     HorizontalScrollbarLayer                    = 1LLU << 16,
     VerticalScrollbarLayer                      = 1LLU << 17,
     PainterForScrollbar                         = 1LLU << 18,
+    MouseDidEnterScrollbar                      = 1LLU << 19,
```

Comment:
MouseIsScrollbarHoverState? Or just `ScrollbarHoverState`

Reply:
Changed to ScrollbarHoverState.